site/api/python/index.html

```diff
@@ -1125,653 +1058,649 @@ <h1 class="title"><span class="name">pulsar</span> module</h1>
+client = pulsar.Client('pulsar://localhost:6650')
 
-<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span><span class="mi">10</span><span class="p">):</span>
-    <span class="n">producer</span><span class="o">.</span><span class="n">send</span><span class="p">(</span><span class="s1">&#39;Hello-</span><span class="si">%d</span><span class="s1">&#39;</span> <span class="o">%</span> <span class="n">i</span><span class="p">)</span>
-
-<span class="n">client</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>
-</pre></div>
+producer = client.create_producer(
+                'persistent://sample/standalone/ns/my-topic')
 
+for i in range(10):
+    producer.send('Hello-%d' % i)
 
+client.close()
+</code></pre>
 <h4><a href="#pulsar.Consumer">Consumer</a> Example</h4>
-<div class="codehilite"><pre><span></span><span class="kn">import</span> <span class="nn">pulsar</span>
-
-<span class="n">client</span> <span class="o">=</span> <span class="n">pulsar</span><span class="o">.</span><span class="n">Client</span><span class="p">(</span><span class="s1">&#39;pulsar://localhost:6650&#39;</span><span class="p">)</span>
-<span class="n">consumer</span> <span class="o">=</span> <span class="n">client</span><span class="o">.</span><span class="n">subscribe</span><span class="p">(</span>
-        <span class="s1">&#39;persistent://sample/standalone/ns/my-topic&#39;</span><span class="p">,</span>
-        <span class="s1">&#39;my-sub&#39;</span><span class="p">)</span>
-
-<span class="k">while</span> <span class="bp">True</span><span class="p">:</span>
-    <span class="n">msg</span> <span class="o">=</span> <span class="n">consumer</span><span class="o">.</span><span class="n">receive</span><span class="p">()</span>
-    <span class="k">print</span><span class="p">(</span><span class="s2">&quot;Received message &#39;</span><span class="si">%s</span><span class="s2">&#39; id=&#39;</span><span class="si">%s</span><span class="s2">&#39;&quot;</span><span class="p">,</span> <span class="n">msg</span><span class="o">.</span><span class="n">data</span><span class="p">(),</span> <span class="n">msg</span><span class="o">.</span><span class="n">message_id</span><span class="p">())</span>
-    <span class="n">consumer</span><span class="o">.</span><span class="n">acknowledge</span><span class="p">(</span><span class="n">msg</span><span class="p">)</span>
+<pre><code>#!python
+import pulsar
 
-<span class="n">client</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>
-</pre></div>
+client = pulsar.Client('pulsar://localhost:6650')
+consumer = client.subscribe(
+        'persistent://sample/standalone/ns/my-topic',
+        'my-sub')
 
+while True:
+    msg = consumer.receive()
+    print("Received message '%s' id='%s'", msg.data(), msg.message_id())
```

Comment:
This `index.html` file needs to be removed. It's already in `.gitignore` and it now gets generated during the build.